src/reps/rep-utils.js

```diff
@@ -167,5 +230,8 @@ module.exports = {
   parseURLParams,
   parseURLEncodedText,
   getFileName,
-  getURLDisplayString
+  getURLDisplayString,
+  isGripSelectableInInspector,
+  getFlattenedGrips,
```

Comment:
I can't actually find where these new methods are being used ?
Is this supposed to be an external API used by some tools not in this repo 
(hey I wouldn't have to ask this in a mono repo setup :) )